apps/api/src/server/applications/applications.routes.js

```diff
@@ -142,4 +143,25 @@ router.get(
 	asyncHandler(getApplicationDetails)
 );
 
+router.patch(
+	'/:id/publish',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/applications/{id}/publish'
+        #swagger.description = 'publish application'
+        #swagger.parameters['id'] = {
+            in: 'path',
+			description: 'Application ID',
+			required: true,
+			type: 'integer'
+		}
+        #swagger.responses[200] = {
+            description: 'response will have the date that the case was published',
+            schema: { publishedDate }
```

Comment:
this does not display any response info in the swagger api, I would suggest:

#swagger.responses[200] = {
            description: 'response will have the date that the case was published as a timestamp',
            schema: { "publishedDate": 1673873105 }
        }